tensorflow_io/ignite/python/tests/ignite_dataset_test.py

```diff
@@ -66,7 +67,7 @@ def _check_dataset(self, dataset):
     self.assertEqual(dtypes.string, dataset.output_types["val"]["NAME"])
     self.assertEqual(dtypes.int64, dataset.output_types["val"]["VAL"])
 
-    it = dataset.make_one_shot_iterator()
+    it = dataset_ops.make_one_shot_iterator(dataset)
```

Comment:
Would be great to update README.md as well. We can do it in the same PR (you can take [this version](https://github.com/tensorflow/tensorflow/blob/4afa2ba93dd606fd3715ef5693dee86fc25e0c77/tensorflow/contrib/ignite/README.md) as a reference) or I can do it in a separate PR.